chart/values.yaml

```diff
@@ -99,6 +105,50 @@ resources: { }
 #   cpu: 100m
 #   memory: 128Mi
 
+deployment:
+  # -- Readiness Probes
+  readinessProbe:
+    failureThreshold: 3
+    periodSeconds: 5
+    successThreshold: 1
+    timeoutSeconds: 10
+    initialDelaySeconds: 30
+    httpGet:
+      path: /_health
+      port: http
+
+  # -- Liveness Probes
+  livenessProbe:
+    periodSeconds: 10
+    failureThreshold: 3
+    successThreshold: 1
+    timeoutSeconds: 10
+    initialDelaySeconds: 15
+    httpGet:
+      path: /_health
+      port: http
+
+  # -- Hardening security
+  securityContext:
+    allowPrivilegeEscalation: false
+    capabilities:
+      drop:
+      - ALL
+    privileged: false
+    runAsGroup: 1000
+    runAsUser: 1000
```

Comment:
The Dockerfile runs with user 1001 https://github.com/bakito/sealed-secrets-web/blob/main/Dockerfile#L32
Can you change this also to the same value?

Reply:
addressed, thanks